wordpress/docker/confd/templates/wp-config.php.tmpl

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * The base configurations of the WordPress.
+ *
+ * This file has the following configurations: MySQL settings, Table Prefix,
+ * Secret Keys, and ABSPATH. You can find more information by visiting
+ * {@link https://codex.wordpress.org/Editing_wp-config.php Editing wp-config.php}
+ * Codex page. You can get the MySQL settings from your web host.
+ *
+ * This file is used by the wp-config.php creation script during the
+ * installation. You don't have to use the web site, you can just copy this file
+ * to "wp-config.php" and fill in the values.
+ *
+ * @package WordPress
+ */
+
+// ** MySQL settings - You can get this info from your web host ** //
+/** The name of the database for WordPress */
+define('DB_NAME', '{{getv "/database/database"}}');
+
+/** MySQL database username */
+define('DB_USER', '{{getv "/database/user"}}');
+
+/** MySQL database password */
+define('DB_PASSWORD', '{{getv "/database/password"}}');
+
+/** MySQL hostname */
+define('DB_HOST', '{{getv "/database/host"}}');
+
+/** Database Charset to use in creating database tables. */
+define('DB_CHARSET', '{{getv "/database/charset"}}');
+
+/** The Database Collate type. Don't change this if in doubt. */
+define('DB_COLLATE', '');
+
+/**#@+
+ * Authentication Unique Keys and Salts.
+ *
+ * Change these to different unique phrases!
+ * You can generate these using the {@link https://api.wordpress.org/secret-key/1.1/salt/ WordPress.org secret-key service}
+ * You can change these at any point in time to invalidate all existing cookies. This will force all users to have to log in again.
+ *
+ * @since 2.6.0
+ */
+define('AUTH_KEY',         'THUTEcuquwe7reth');
+define('SECURE_AUTH_KEY',  'sWApuqa53aKuC7aQ');
+define('LOGGED_IN_KEY',    'muqAdAsEFrAb5d77');
+define('NONCE_KEY',        '4UstarUkaCErEHaB');
+define('AUTH_SALT',        'vucrapRub4ew5sPe');
+define('SECURE_AUTH_SALT', 'sajABrAmaxa8Ra3h');
+define('LOGGED_IN_SALT',   'Pr7pHazuPucHacuD');
+define('NONCE_SALT',       'prePhatrahEPhu3W');
+
+/**#@-*/
+
+/**
+ * WordPress Database Table prefix.
+ *
+ * You can have multiple installations in one database if you give each a unique
+ * prefix. Only numbers, letters, and underscores please!
+ */
+$table_prefix  = 'wp_';
```

Comment:
A security thing is to have a different prefix than the standard wp_ (can be anything)